remote/remote_test.go

```diff
@@ -501,6 +503,77 @@ func testImage(t *testing.T, when spec.G, it spec.S) {
 				})
 			})
 		})
+
+		when("#RetryLogic", func() {
+			var server *httptest.Server
+			var repo string
+
+			when("Manifest API in registry returns status code 200", func() {
+				repo = "org/do-not-retry"
+				mockServer := h.NewMockServer(repo, http.StatusOK, 0)
```

Comment:
Why aren't these lines under the `it.Before`?

Reply:
If you'll move it to be under `it.Before`, you won't need line 509 but `mockServer` will need to be defined there instead.

Reply:
Yes, I just noticed that

Reply:
I managed to simplify the variable declaration and only the MockServer was required.